Lib/fontTools/varLib/mutator.py

```diff
@@ -63,11 +64,11 @@ def instantiateVariableFont(varfont, location, inplace=False):
 	glyf = varfont['glyf']
 	# get list of glyph names in gvar sorted by component depth
 	glyphnames = sorted(
-		gvar.variations.keys(),
-		key=lambda name: (
-			glyf[name].getCompositeMaxpValues(glyf).maxComponentDepth
-			if glyf[name].isComposite() else 0,
-			name))
+			gvar.variations.keys(),
```

Comment:
Are these whitespace changes intentional, or your editor settings do that?

Reply:
I think its because of my editor settings. Let me check how to revert them.

Reply:
Reverted them.